packages/story-editor/src/output/test/page.js

```diff
@@ -1243,6 +1243,41 @@ describe('Page output', () => {
         'background-audio="https://example.com/audio.mp3"'
       );
     });
+    it('should add background audio as amp-video', async () => {
```

Comment:
An AMP validation test for background audio would be good

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/10288/commits/37c26ccee02ec3c2c449ea3e626cc6772d35bd58